support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -88,7 +100,7 @@ function getParticipationsFromUrl(): Participations | null | undefined {
 
 	if (hashUrl.startsWith('#ab-')) {
 		const [testId, variant] = decodeURI(hashUrl.substr(4)).split('=');
-		const test = {};
+		const test: Record<string, string> = {};
```

Comment:
Not necessary, but I wonder if this would be more straightforward as:

```ts
return { [testId]: variant };
```

Reply:
Yeah this would be a nice simplification